test/relations.test.js

```diff
@@ -90,6 +91,18 @@ describe('relations', function () {
         });
       });
 
+      it('should create record on scope with promises', function (done) {
+        Book.create()
+        .then (function (book) {
+          return book.chapters.create()
+          .then (function (c) {
```

Comment:
Coding style: indent `.then` by two spaces, no space before opening bracket. Please apply the fix everywhere in the test file.

```js
return book.chapters.create()
  .then(function(c) {
    //...
  })
  .catch(done);
```